analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/PublicMethodArgumentsShouldBeCheckedForNull.CSharp11.cs

```diff
@@ -25,3 +25,11 @@ public static virtual void NotCompliantCases(object o)
         o.ToString(); // FIXME Non-compliant {{Refactor this method to add validation of parameter 'o' before using it.}}
     }
 }
+
+file class ClassWithFileAccessibility
```

Comment:
Given our current logic, this needs to be an interface to cover this line
`|| (containingType is InterfaceDeclarationSyntax && HasNoDeclaredAccessibilityModifier(modifiers)` or do I miss something?
You should add a "file" scoped interface with a member implementation as well.